src/sbt-test/travisci/sbt-versions/build.sbt

```diff
@@ -0,0 +1,8 @@
+def assertEq[A](left: A, right: A) = assert(left == right, s"$left != $right")
+
+TaskKey[Unit]("check") := {
+  assertEq(sbtVersion.value, "1.2.0")
+  assertEq((crossSbtVersions in ThisBuild).value, Seq("0.13.17", "1.2.0"))
+  // This fails since crossSbtVerions.value is Vector("1.2.0") 
+  // assertEq((crossSbtVersions).value, Seq("0.13.17", "1.2.0"))
```

Comment:
I suspect this comment-outed assertion fails due to [this line in sbt](https://github.com/sbt/sbt/blob/ea778e9a5ce12b72bb9efe84bf364d0997f4632c/main/src/main/scala/sbt/Defaults.scala#L483)
But I have no idea how to fix this in `sbt-travisci`.

Reply:
The only way to solve this would be by redefining it in project settings but that should only be done as an opt-in AutoPlugin, rather than an auto-triggering AutoPlugin.